### PR TITLE
Find match count (rebase of #5051)

### LINKED ISSUE #4805: Show a count # of matches when searching for text in a PDF
When searching for a word or text in a PDF using the "Find in Document" button, show the count # of matches found in the little search pop-up to the right of the "Match Case" checkbox.

### PR DESCRIPTION
**Pull request by yurydelendik**
Bases of #5051

Fixes #4805

### CODE REVIEW

File: web/pdf_find_controller.js [hunk 1]
```diff
@@ -386,6 +395,15 @@ var PDFFindController = (function PDFFindControllerClosure() {
       }
     },
 
+    updateUIResultCount:
+        function PDFFindController_updateUICountMatches() {
```
Comment (Snuffleupagus) [inline web/pdf_find_controller.js]: Nit: The function in `pdf_find_bar.js` is called `updateResultsCount`, so for consistency we might want to add an "s" to the current method name, i.e. `updateUIResultsCount` instead!?.
(Also, the name after `function` isn't entirely consistent.)

**Comment by Snuffleupagus** [comment]
r=me, with nit addressed.

**Comment by Rob--W** [comment]
Is it feasible to add an index to the counter, e.g. "1 out of 1"? And also, a tooltip that shows what the number means when hovering over the number may be helpful.

**Comment by timvandermeij** [comment]
Could you open a separate issue for that and label it as a feature?

**Pull request merged**